circle.yml

```diff
@@ -13,4 +13,4 @@ deployment:
       - git config --global user.name "Apiary Bot"
       - git config --global user.email "support@apiary.io"
       - git remote set-url origin "https://${GH_TOKEN}@github.com/apiaryio/apiaryio.github.com"
-      - rake deploy
+      - bundle exec rake deploy
```

Comment:
Ah right I see. I think a different approach would be better.

At the top of the Rakefile we can add the following:

```ruby
require 'bundler'
Bundler.setup(:default)
```

This means that any require's after will go through bundler.